backend/de.metas.swat/de.metas.swat.base/src/main/java/de/metas/invoicecandidate/process/C_Invoice_Candidate_EnqueueSelectionForInvoicingAndPDFPrinting.java

```diff
@@ -0,0 +1,157 @@
+/**
+ *
+ */
+package de.metas.invoicecandidate.process;
+
+/*
+ * #%L
+ * de.metas.swat.base
+ * %%
+ * Copyright (C) 2015 metas GmbH
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation, either version 2 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program. If not, see
+ * <http://www.gnu.org/licenses/gpl-2.0.html>.
+ * #L%
+ */
+
+import de.metas.adempiere.form.IClientUI;
+import de.metas.async.AsyncBatchId;
+import de.metas.async.api.IAsyncBatchBL;
+import de.metas.async.api.impl.AsyncBatchDAO;
+import de.metas.async.model.I_C_Async_Batch;
+import de.metas.i18n.IMsgBL;
+import de.metas.invoicecandidate.api.IInvoiceCandBL;
+import de.metas.invoicecandidate.api.IInvoiceCandidateEnqueueResult;
+import de.metas.invoicecandidate.api.IInvoiceCandidateEnqueuer;
+import de.metas.invoicecandidate.api.IInvoicingParams;
+import de.metas.invoicecandidate.api.impl.InvoicingParams;
+import de.metas.invoicecandidate.model.I_C_Invoice_Candidate;
+import de.metas.process.IProcessPrecondition;
+import de.metas.process.IProcessPreconditionsContext;
+import de.metas.process.JavaProcess;
+import de.metas.process.PInstanceId;
+import de.metas.process.ProcessExecutionResult.ShowProcessLogs;
+import de.metas.process.ProcessPreconditionsResolution;
+import de.metas.process.ProcessPreconditionsResolution.ProcessCaptionMapper;
+import de.metas.process.RunOutOfTrx;
+import de.metas.security.permissions.Access;
+import de.metas.util.Check;
+import de.metas.util.Services;
+import lombok.NonNull;
+import org.adempiere.ad.dao.IQueryBL;
+import org.adempiere.ad.dao.IQueryBuilder;
+import org.adempiere.ad.dao.IQueryFilter;
+import org.adempiere.ad.trx.api.ITrx;
+import org.adempiere.exceptions.AdempiereException;
+import org.adempiere.util.api.IParams;
+import org.compiere.SpringContextHolder;
+import org.compiere.model.IQuery;
+import org.compiere.util.DB;
+import org.compiere.util.Ini;
+
+import javax.annotation.Nullable;
+import java.math.BigDecimal;
+import java.util.Properties;
+
+import static de.metas.async.Async_Constants.C_Async_Batch_InternalName_InvoiceCandidate_Processing;
+
+public class C_Invoice_Candidate_EnqueueSelectionForInvoicingAndPDFPrinting extends JavaProcess
```

Comment:
What's the differenence between this process and the "normal" enqueing processes?
`C_Async_Batch_InternalName_InvoiceCandidate_Processing` is used elsewhere, too...so when is the auto-concatenating of PDFs performed?